source/rules/no-tap.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * @license Use of this source code is governed by an MIT-style license that
+ * can be found in the LICENSE file at https://github.com/cartant/eslint-plugin-rxjs
+ */
+
+import { Rule } from "eslint";
+import * as es from "estree";
+
+const rule: Rule.RuleModule = {
+  meta: {
+    docs: {
+      category: "RxJS",
+      description: "Disallows the use of the tap operator.",
+      recommended: false
+    },
+    fixable: null,
+    messages: {
+      forbidden: "The `tap` operator is forbidden."
+    },
+    schema: []
+  },
+  create: context => {
+    return {
+      "ImportDeclaration[source.value='rxjs/operators'] > ImportSpecifier[imported.name='tap']": (
```

Comment:
Have to check on `imported`, otherwise `tap` in the following scenario would be reported twice (once as imported, and once as local)

`import { tap } from 'rxjs/operators'`